doc/contributing/contributing-to-documentation.chapter.md

```diff
@@ -1,23 +1,12 @@
 # Contributing to this documentation {#chap-contributing}
 
-The sources of the Nixpkgs manual are in the [doc](https://github.com/NixOS/nixpkgs/tree/master/doc) subdirectory of the Nixpkgs repository. The manual is still partially written in DocBook but it is progressively being converted to [Markdown](#sec-contributing-markup).
-
-You can quickly check your edits with `make`:
-
-```ShellSession
-$ cd /path/to/nixpkgs/doc
-$ nix-shell
-[nix-shell]$ make
-```
-
-If you experience problems, run `make debug` to help understand the docbook errors.
+The sources of the Nixpkgs manual are in the [doc](https://github.com/NixOS/nixpkgs/tree/master/doc) subdirectory of the Nixpkgs repository.
 
 After making modifications to the manual, it's important to build it before committing. You can do that as follows:
 
 ```ShellSession
 $ cd /path/to/nixpkgs/doc
 $ nix-shell
-[nix-shell]$ make clean
 [nix-shell]$ nix-build .
```

Comment:
`nix-shell` call is no longer needed.

Maybe we could just use `nix-build ./doc`.